src/base/projects/Browser.svelte

```diff
@@ -21,6 +21,7 @@
   export let config: Config;
   export let org: string | null = null;
   export let user: string | null = null;
+  export let seed: string | null = null;
```

Comment:
Why are we using a fallback to `null` here, it seems that neither in `View` or `ProjectContentRoutes` we could encounter us with a missing org, user or seed, since they are `""` from the beginning

Reply:
hmm good question, will look into this.